src/main/java/Duke.java

```diff
@@ -1,10 +1,93 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+    enum TaskList {
+        TODO, DEADLINE, EVENT
+    }
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
+        Scanner scanner = new Scanner(System.in);
+        /*String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello from\n" + logo);*/
+        System.out.println("Hello! I'm Duke\nWhat can I do for you?");
+        ArrayList<Task> todo = new ArrayList<>();
```

Comment:
You might want to consider renaming todo since the ArrayList does not contain only Todo, naming convention recommends plural form too!

Reply:
I agree with Permas, it might confuse you later on as well.